content/en/docs/instrumentation/net/automatic/config.md

```diff
@@ -0,0 +1,352 @@
+| -------------------------------- | ---------------------------------------------------------------------------- | ------------- | --------------------------------------------------- |
+| `OTEL_BSP_SCHEDULE_DELAY`        | Delay interval (in milliseconds) between two consecutive exports.            | `5000`        | [Stable](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_BSP_EXPORT_TIMEOUT`        | Maximum allowed time (in milliseconds) to export data                        | `30000`       | [Stable](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_BSP_MAX_QUEUE_SIZE`        | Maximum queue size.                                                          | `2048`        | [Stable](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_BSP_MAX_EXPORT_BATCH_SIZE` | Maximum batch size. Must be less than or equal to `OTEL_BSP_MAX_QUEUE_SIZE`. | `512`         | [Stable](/docs/specs/otel/versioning-and-stability) |
+
+### Metrics exporter
+
+| Environment variable          | Description                                                                   | Default value                                           | Status                                              |
+| ----------------------------- | ----------------------------------------------------------------------------- | ------------------------------------------------------- | --------------------------------------------------- |
+| `OTEL_METRIC_EXPORT_INTERVAL` | The time interval (in milliseconds) between the start of two export attempts. | `60000` for OTLP exporter, `10000` for console exporter | [Stable](/docs/specs/otel/versioning-and-stability) |
+| `OTEL_METRIC_EXPORT_TIMEOUT`  | Maximum allowed time (in milliseconds) to export data.                        | `30000` for OTLP exporter, none for console exporter    | [Stable](/docs/specs/otel/versioning-and-stability) |
+
+### Logs exporter
+
+| Environment variable                              | Description                                             | Default value | Status                                                    |
+| ------------------------------------------------- | ------------------------------------------------------- | ------------- | --------------------------------------------------------- |
+| `OTEL_DOTNET_AUTO_LOGS_INCLUDE_FORMATTED_MESSAGE` | Whether the formatted log message should be set or not. | `false`       | [Experimental](/docs/specs/otel/versioning-and-stability) |
+
+### OTLP
+
+**Status**: [Stable](/docs/specs/otel/versioning-and-stability)
+
+To enable the OTLP exporter, set the
+`OTEL_TRACES_EXPORTER`/`OTEL_METRICS_EXPORTER`/`OTEL_LOGS_EXPORTER` environment
+variable to `otlp`.
+
+To customize the OTLP exporter using environment variables, see the
+[OTLP exporter documentation](https://github.com/open-telemetry/opentelemetry-dotnet/tree/core-1.5.1/src/OpenTelemetry.Exporter.OpenTelemetryProtocol#environment-variables).
```

Comment:
is there a reason that this is locked to core-1.5.1? If the version needs to be fixed we should move it to a parameter that can be updated by a bot eventually like we do it for java

Reply:
I was wondering the same. @pjanotti @Kielek what do you reckon?

Reply:
Just to be sure, that there were no additional functionalities included in vNext version, not included in the version used by OTel Auto.

Reply:
Yes, this should move together with the version of .NET auto-instrumentation.